src/classes/api_client.py

```
@@ -596,6 +615,49 @@ def search(
+            "file_id": file_id,
+            "index_name": index_name,
+            "filename": filename or file_id,
+        }
+        if existing_index_id:
+            payload["existing_index_id"] = existing_index_id
+        try:
+            s = session or self.session
+            r = s.post(f"{self.api_url}/indexing", json=payload, timeout=30)
+            r.raise_for_status()
+            data = r.json()
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)